scripts/pre_commit_linter.py

```diff
@@ -225,6 +235,17 @@ def _pre_commit_linter():
             print 'Could not locate file or directory %s. Exiting.' % input_path
             print '----------------------------------------'
             sys.exit(0)
+    if parsed_args.files:
+        file_list = []
+        invalids = []
```

Comment:
I'd suggest calling these valid_filepaths and invalid_filepaths. (This relates the two names better to each other, and I think 'filepaths' is more accurate than 'files'.)